catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/CachingFederationStrategy.java

```diff
@@ -193,7 +198,9 @@ public QueryResponse federate(List<Source> sources, QueryRequest queryRequest) {
     }
 
     QueryResponse queryCache(QueryRequest queryRequest) {
-        QueryRequest updatedRequest = getRequestWithTagsFilter(queryRequest);
+        QueryRequest updatedRequest = validationQueryFactory.getQueryRequestWithValidationFilter(
```

Comment:
I looked at it and a good way to refactor wasn't readily apparent. I thought about putting it in the `federate` method, since that calls both `queryCache` and `sourceFederate` but it looks like `sourceFederate` should only modify the queryRequest in a specific case